[Compilation Lost compatibility with Qt5

### Problem description

Cannot build with Qt5.15 due to the use of `QList.emplace_back()` in TechDraw:
```
[ 85%] Building CXX object src/Mod/TechDraw/Gui/CMakeFiles/TechDrawGui.dir/QGIProjGroup.cpp.o
/home/gael/04-sources/freecad/FreeCAD/src/Mod/TechDraw/Gui/QGIProjGroup.cpp: In member function ‘QList<TechDrawGui::QGIViewPart*> TechDrawGui::QGIProjGroup::secondaryQViews() const’:
/home/gael/04-sources/freecad/FreeCAD/src/Mod/TechDraw/Gui/QGIProjGroup.cpp:249:16: error: ‘class QList<TechDrawGui::QGIViewPart*>’ has no member named ‘emplace_back’
  249 |         result.emplace_back(qview);
      |                ^~~~~~~~~~~~
```

According to https://doc.qt.io/archives/qt-5.15/qlist.html the method is not implemented.

### Workbench affected?

TechDraw

### Steps to reproduce

1. `make`

### Expected behavior

Should compile with Qt5.15.

### Actual behavior

Cf. above error.

### Development version About Info (in Safe Mode)

```shell
The culprit is commit fd014445fbfb.
```

### Last known good version (optional)

```shell
Commit 5d4495785b72?
```

--- a/src/Mod/TechDraw/Gui/QGIProjGroup.cpp
+++ b/src/Mod/TechDraw/Gui/QGIProjGroup.cpp
@@ -246,7 +246,7 @@ QList<QGIViewPart*> QGIProjGroup::secondaryQViews() const
         if (!qview) {
             continue;
         }
-        result.emplace_back(qview);
+        result.append(qview);
     }
     return result;
 }

